app/templates/components/home-page.emblem

```diff
@@ -29,18 +29,18 @@ if isLoaded
           .appknox-logo
             = img-logo
           ul.menu-list.text-center.margin-top20
-            li
-              = link-to 'authenticated.index' class="mp-auth-index"
-                div
-                  = fa-icon "home" class="sidebar-icons"
-                  = t "dashboard"
             li
               = link-to 'authenticated.projects' class="mp-auth-projects"
                 div
                   = fa-icon "tasks" class="sidebar-icons"
                   = t "projects"
                 span.badge-square.is-sidebar.content-right
                   | #{organization.selected.projectsCount}
+            li
+              = link-to 'authenticated.analytics' class="mp-auth-index"
```

Comment:
What's the use of `mp-auth-*` class? shouldn't it be `mp-auth-analytics`?

Reply:
Its for mixpanel